### PR TITLE
ffmpegx: disable nonfree - Tvh and ffmpeg-tools update

### PR DESCRIPTION
**Pull request by CvH**
- gnutls is build without installing to the toolchain to avoid that random packages pick it up

any idea where to place libidn2 ? or maybe all packages should go to addon deps ?

### CODE REVIEW

**Comment by CvH** [comment]
added `xz: add pic for xz` to fix imagemagick building

File: packages/security/nettle/package.mk [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+PKG_SITE="http://www.lysator.liu.se/~nisse/nettle"
+PKG_URL="http://ftpmirror.gnu.org/gnu/nettle/nettle-$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain gmp"
+PKG_LONGDESC="A low-level cryptographic library."
+
+PKG_CONFIGURE_OPTS_TARGET="--disable-documentation \
+                           --disable-openssl"
+
+if target_has_feature neon; then
+  PKG_CONFIGURE_OPTS_TARGET="$PKG_CONFIGURE_OPTS_TARGET --enable-arm-neon"
```
Comment (5schatten) [inline packages/security/nettle/package.mk]: What about:
`PKG_CONFIGURE_OPTS_TARGET+=" --enable-arm-neon"`

**Comment by MilhouseVH** [comment]
OK to merge?

**Comment by MilhouseVH** [comment]
@CvH could you squash this commit into `ffmpegx`: http://ix.io/1xRZ - we shouldn't be trying to build `x265` for non-x86_64 platforms (as the `x265` package is x86_64 only). Thanks.

**Comment by CvH** [comment]
@MilhouseVH added and updated to latest
should be okay to merge I think

**Comment by MilhouseVH** [comment]
Many thanks - let's give it a day or two more after the updates in test builds without further changes then I think it should be ok to merge (assuming no issue reports of course!)

**Pull request merged**